Naming/Polyfill/FileExtensionTrait.php

```diff
@@ -17,11 +17,11 @@ private function getExtension(UploadedFile $file): ?string
     {
         $originalName = $file->getClientOriginalName();
 
-        if ($extension = \pathinfo($originalName, PATHINFO_EXTENSION)) {
+        if ('' !== ($extension = \pathinfo($originalName, PATHINFO_EXTENSION))) {
```

Comment:
I'd rather check for `null`

Reply:
Hm… I'm unsure about that.

The [php docs](https://www.php.net/manual/en/function.pathinfo.php#refsect1-function.pathinfo-returnvalues) says 

> If options is present, returns a string containing the requested element. 

and testing a file without extension `var_dump(\pathinfo('foo/bar', PATHINFO_EXTENSION));"` shows `string(0) ""`.

Reply:
You're right. So what is the purpose of checking with `is_string` below?